routes/cube_routes.js

```diff
@@ -416,32 +415,30 @@ router.get('/overview/:id', async (req, res) => {
 
     // calc cube prices
     for (const card of cube.cards) {
-      card.details = { ...carddb.cardFromId(card.cardID, 'name tcgplayer_id') };
+      card.details = { ...carddb.cardFromId(card.cardID) };
```

Comment:
```suggestion
      card.details = { ...carddb.cardFromId(card.cardID), 'name prices' };
```
Not filtering fields here leads to much bigger page size. There might be some others that are needed.